frontend/device/android/powerd.lua

```diff
@@ -20,7 +21,7 @@ function AndroidPowerD:frontlightIntensityHW()
 end
 
 function AndroidPowerD:setIntensityHW(intensity)
-    -- if frontlight switch was toggled of, turn it on
+    -- if frontlight switch was toggled off, turn it on
```

Comment:
If you're going to modify the line, try to make the comments proper sentences. :angel: 
```suggestion
    -- If the frontlight switch was off, turn it on.
```

Reply:
OK :D